pytest_redis/factories/client_async.py

```diff
@@ -0,0 +1,65 @@
+"""Redis async client fixture factory."""
+
+from typing import AsyncGenerator, Callable, Literal, Optional, Union
+
+import pytest
+from _pytest.fixtures import FixtureRequest
+from redis.asyncio import Redis
+
+from pytest_redis.config import get_config
+from pytest_redis.executor import NoopRedis, RedisExecutor
+
+
+def redisdb_async(
+    process_fixture_name: str, dbnum: int = 0, decode: Optional[bool] = None
+) -> Callable[[FixtureRequest], AsyncGenerator[Redis]]:
+    """Create connection fixture factory for pytest-redis.
+
+    :param process_fixture_name: name of the process fixture
+    :param dbnum: number of database to use
+    :param decode: Client: to decode response or not.
+        See redis.StrictRedis decode_reponse client parameter.
+    :returns: function which makes a connection to redis
+    """
+
+    @pytest.fixture
+    async def async_redisdb_factory(request: FixtureRequest) -> AsyncGenerator[Redis]:
```

Comment:
Question, As I haven't done pytest-async yet.

Do we have to have pytest-async installed in order for it to work?

Reply:
Also, what at version asyncio got introduced in redis-py?

Reply:
> Do we have to have pytest-async installed in order for it to work?

Well it is not mandatory but `pytest-asyncio` has been a standard way to use async fixtures and run async tests with Pytest.

> Also, what at version asyncio got introduced in redis-py?

Very good question, I would say a long long time already. (I didn't check all the way out but commit history on `redis-py` has asyncio package before 2022 :)